### PR TITLE
Added avatar component

### PR DESCRIPTION
**Pull request by poojavishnoi**

### CODE REVIEW

File: components/Avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
+@import url('https://fonts.googleapis.com/css2?family=Abril+Fatface&family=Bebas+Neue&family=IBM+Plex+Serif:wght@300&family=Lora&family=Merriweather:wght@300&family=Montserrat:wght@300&family=Roboto:wght@300&display=swap');
+
+:root {
+  --primary-color: #FF6D6D;
+}
```
Comment (yashpatel1797) [inline components/Avatar/avatar.css:5]: add root in common CSS to avoid repeating code.

File: doc/docstyle.css [hunk 2]
```diff
@@ -0,0 +1,431 @@
...
+  justify-content: space-between;
+  padding: 1rem;
+}
+
+.header_links_container {
+  margin-left: auto;
+}
+
+.header a {
+  color: white;
```
Comment (yashpatel1797) [inline doc/docstyle.css]: try to use var for white also.

File: doc/docstyle.css [hunk 3]
```diff
@@ -0,0 +1,431 @@
...
+
+.primary_alert {
+  background-color: #ff7474;
+  color: white;
+}
+
+.primary_alert a:hover {
+  border-radius: 20px;
+  transition: 0.4s ease;
+  background-color: rgb(214, 125, 125);
```
Comment (yashpatel1797) [inline doc/docstyle.css]: Maintain consistency in using colour if following # method then follows everywhere.

**Review by yashpatel1797 — CHANGES_REQUESTED** [review]
I have a few suggestions
1. can create one common CSS file where import and root can be added.

File: doc/docstyle.css (hunk 3 above)
Comment (poojavishnoi) [reply doc/docstyle.css]: Okay..Great suggestions:)

File: doc/docstyle.css (hunk 2 above)
Comment (poojavishnoi) [reply doc/docstyle.css]: okay

File: components/Avatar/avatar.css (hunk 1 above)
Comment (poojavishnoi) [reply components/Avatar/avatar.css:5]: yup okay.

**Pull request merged**